src/ducks/transactions/actions/HealthExpenseStatusAction.jsx

```diff
@@ -93,14 +93,16 @@ const action = {
 
     return <Icon icon={hourglassIcon} color={color} />
   },
-  match: (transaction, { brands }) => {
-    const allBrandsHealth = allBrands.filter(brand => brand.health)
-    const brandsHealth = brands.filter(brand => brand.health)
+  match: (
+    transaction
+    /* { brands } */
+  ) => {
+    // const allBrandsHealth = allBrands.filter(brand => brand.health)
+    // const brandsHealth = brands.filter(brand => brand.health)
 
     return (
-      allBrandsHealth.length > brandsHealth.length &&
-      isHealthExpense(transaction) &&
-      getVendors(transaction)
+      // allBrandsHealth.length > brandsHealth.length && // TODO why this test ??
```

Comment:
Did you find an answer to the question "why this test ??" ? :)

Reply:
No ! Good catch :) @kosssi any idea ?